### PR TITLE
Bs vars

### PR DESCRIPTION
**Pull request by samsonasu**

### CODE REVIEW

File: lib/subspace/cli.rb [hunk 1]
```diff
@@ -62,8 +62,9 @@ def run
     command :vars do |c, args|
       c.syntax = 'subspace vars [environment]'
       c.summary = 'View or edit the encrypted variables for an environment'
-      c.description = ''
+      c.description = 'You can call subspace vars'
```
Comment (petitJAM) [inline lib/subspace/cli.rb]: when does a user see this?  this doesn't sound right

File: ansible/playbooks/local_template.retry [hunk 2]
```diff
@@ -0,0 +1 @@
+localhost
```
Comment (boatrite) [inline ansible/playbooks/local_template.retry]: retry files should be gitignored

File: template/provision/templates/application.yml.template [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# These environment variables are applied to all environments
+SECRET_KEY_BASE: {{secret_key_base}}
+
+development:
```
Comment (petitJAM) [inline template/provision/templates/application.yml.template]: does this generate the app yml for me locally? how does that work

Comment (petitJAM) [inline template/provision/templates/application.yml.template]: `subspace provision development`?

File: template/provision/templates/application.yml.template [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# These environment variables are applied to all environments
+SECRET_KEY_BASE: {{secret_key_base}}
```
Comment (boatrite) [inline template/provision/templates/application.yml.template]: Shouldn't this be different for each environment?

Comment (samsonasu) [inline template/provision/templates/application.yml.template]: yes, and it is -- this pulls from the ansible variables in the vault which can be defined differently for different environents

Comment (petitJAM) [inline template/provision/templates/application.yml.template]: the wording could definitely be confusing to someone new to this
```# These environment variables are applied to all environments```
kinda makes it sound like it's the same value across all environments

File: template/provision/templates/application.yml.template (hunk 3 above)
Comment (samsonasu) [inline template/provision/templates/application.yml.template]: `subspace vars development --create`

**Pull request merged**